pkg/operator/starter.go

```diff
@@ -160,7 +155,13 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	if err != nil {
 		return err
 	}
-	migrator := migrators.NewInProcessMigrator(dynamicClientForMigration, kubeClient.Discovery())
+
+	migrationClientConfig := dynamic.ConfigFor(controllerContext.KubeConfig)
+	migrationClientConfig.Burst = 40
+	migrationClientConfig.QPS = 30
```

Comment:
this client is just for the CRD. No need to tweak the values. Instead we have to verify that kube-storage-version-migrator does the same tweaking internally.